### DISCUSSION TITLE
Add a clear input button to the search box

### DISCUSSION THREAD

**Discussion by vadrile** [ppy/osu#31069] [Ideas]
I understand that pressing the Esc key works, but I think it would be more convenient if everything could be done using only the mouse.
I think this will also address this issue to some extent: https://github.com/ppy/osu/discussions/24470
(I use machine translation.)
![スクリーンショット 2024-12-11 081151](https://github.com/user-attachments/assets/0473866e-c24b-44ab-bbfa-00ba9d86f1ac)

**Comment by peppy** [comment]
Definitely something we can add to all search boxes game-wide
